translations/messages+intl-icu.de.yml

```
@@ -964,7 +964,11 @@ elements.bulk.edit.change.state.hint: "Planungsdokumente, die bereits den hier g
 elements.bulk.edit.change.state.success: "Die automatische Umstellung wurde bei <strong>{changed}</strong> von <strong>{total}</strong> Planungsdokumenten erfolgreich aktiviert."
 elements.create.subcategory: "Neue Unterkategorie erstellen"
 elements.export: "Planungsdokumente herunterladen"
-elements.selected: "Elemente ausgewählt"
+elements.selected: |
+    {count, plural,
+        =1 { Element ausgewählt }
+        other { Elemente ausgewählt }
```

We actually have to add the number. See https://github.com/demos-europe/demosplan-core/blob/40ee03ff06d5986bf4568f448c6185aa0204af26/translations/messages+intl-icu.de.yml#L329

```suggestion
        =1 { Ein Element ausgewählt }
        other { # Elemente ausgewählt }
```

We also have to change `items.selected.multi.page` the same way.

---

We also have to change `items.selected.multi.page the same way. Currently the number is not included there.

---

Done in https://github.com/demos-europe/demosplan-core/pull/2245/commits/1433d78d53d6737467aa6672293018b9c1987f55